napalm_yang/mappings/eos/parsers/state/openconfig-platform/components.yaml

```diff
@@ -0,0 +1,75 @@
+---
+metadata:
+    parser: JSONParser
+    execute:
+        - method: "device.run_commands"
+          args:
+              commands: ["show inventory"]
+
+components:
+    _parse: unnecessary
+    component:
+        _parse:
+            mode: dict
+            from: "{%- set data = {} -%}{%- for k, v in bookmarks.components.0.0.items() -%}{%- if v is mapping and k is not equalto 'systemInformation' -%}{%- for kk, vv in v.items() -%}{%- do data.update({'{}_{}'.format(k,kk): vv}) -%} {%- endfor -%} {%- endif -%} {%- endfor -%} {{ data|json }}"
```

Comment:
````python
_parse_leaf_transform(cls, mapping):
```
Have a mapping["method"] to call?  How would we make sure the method is imported?  Looking at how complex transforming the json into the model is, doing the brunt of the transformation in python might be better.